frontend/src/main/components/Users/UsersTable.js

```diff
@@ -1,10 +1,15 @@
 import OurTable, { ButtonColumn } from "main/components/OurTable";
 import { useBackendMutation } from "main/utils/useBackend";
+import { toast } from "react-toastify";
+import axios from "axios";
+import { useMutation, useQueryClient } from "react-query";
 
 export default function UsersTable({ users }) {
   // Stryker disable all : hard to test for query caching
   // Stryker enable all
```

Comment:
It appears that this team removed the code here, but did not remove the stryker lines.

Since you are in this file anyway, can you clean this up by removing both of these lines?

(This was in #27 )

<img width="1070" alt="image" src="https://github.com/user-attachments/assets/bdbfd99e-40ea-4604-ad16-a16933353c7f" />